libnopegl/src/node_animated.c

```diff
@@ -36,34 +36,44 @@ static const struct node_param animatedtime_params[] = {
     {"keyframes", NGLI_PARAM_TYPE_NODELIST, OFFSET(animkf), .flags=NGLI_PARAM_FLAG_DOT_DISPLAY_PACKED,
                   .node_types=(const uint32_t[]){NGL_NODE_ANIMKEYFRAMEFLOAT, NGLI_NODE_NONE},
                   .desc=NGLI_DOCSTRING("time key frames to interpolate from")},
+    {"time_offset", NGLI_PARAM_TYPE_F64, OFFSET(time_offset),
+                    .desc=NGLI_DOCSTRING("how much the time is shifted before evaluating the animation")},
     {NULL}
 };
 
 static const struct node_param animatedfloat_params[] = {
     {"keyframes", NGLI_PARAM_TYPE_NODELIST, OFFSET(animkf), .flags=NGLI_PARAM_FLAG_DOT_DISPLAY_PACKED,
                   .node_types=(const uint32_t[]){NGL_NODE_ANIMKEYFRAMEFLOAT, NGLI_NODE_NONE},
                   .desc=NGLI_DOCSTRING("float key frames to interpolate from")},
+    {"time_offset", NGLI_PARAM_TYPE_F64, OFFSET(time_offset),
+                    .desc=NGLI_DOCSTRING("how much the time is shifted before evaluating the animation")},
```

Comment:
nit (shorter desc): `apply a time offset before evaluating the animation`

Reply:
sure done